embc-app/embc-app.csproj

```diff
@@ -82,7 +104,28 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Update="SeedData\*.json">
+    <Content Update="Seeder\SeedData\Communities.json">
```

Comment:
If the json files are embedded (which is a great idea), do we need to copy them to the output folder?

Reply:
Fixed the project to not include the files in the output